.github/workflows/tf_apply.yml

```diff
@@ -0,0 +1,53 @@
+name: "Terraform Apply"
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  TERRAFORM_VERSION: 1.0.3
+  TERRAGRUNT_VERSION: v0.31.1
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+  actions: write
+  checks: write
+  statuses: write
+
+jobs:
+  terragrunt-apply:
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: ca-central-1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1.2.1
+        with:
+          terraform_version: ${{ env.TERRAFORM_VERSION }}
+          terraform_wrapper: false
+
+      - name: Setup Terragrunt
+        run: |
+          mkdir -p bin
+          wget -O bin/terragrunt https://github.com/gruntwork-io/terragrunt/releases/download/$TERRAGRUNT_VERSION/terragrunt_linux_amd64
+          chmod +x bin/terragrunt
+          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
```

Comment:
Pat made this super simple now.

https://github.com/cds-snc/scan-files/blob/main/.github/workflows/tf_apply_production.yml#L35-L36

Reply:
I actually wrote that action and promptly forgot about it 😢

Reply:
my first expose was through Pat and assumed he wrote it. Now i know! I'm happy to be able to remind you about it 🎉